ajax/libs/apexcharts/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "apexcharts",
+  "description": "A JavaScript Chart Library",
+  "version": "1.1.3",
+  "filename": "dist/apexcharts.min.js",
```

Comment:
`filename` should point to the main with the relative path to the `basePath` in auto-update config.